bsoid_app/machine_learner.py

```diff
@@ -9,14 +9,15 @@
 from bsoid_app.bsoid_utilities import visuals
 from bsoid_app.bsoid_utilities.load_workspace import load_classifier
 from streamlit import caching
-
-
+import matplotlib.pyplot as plt
+        
+        
 class Protocol:
-
-    def __init__(self, working_dir, prefix, features, sampled_features, assignments):
-        st.subheader('CREATE A MODEL')
-        self.working_dir = working_dir
-        self.prefix = prefix
+        
```

Comment:
On VSCode, hit `Command + ,` to open settings. Search for `Files: Trim Trailing Whitespace` and enable it